popularity/.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: dart
```

Comment:
Since you have modified `.travis.yml` above, I don't think we need this file.

Reply:
This file is used by `mono_repo` to generate the root `.travis.yml` and the associated script.